WordPress/Classes/Stores/RemoteFeatureFlagStore.swift

```diff
@@ -0,0 +1,93 @@
+import Foundation
+import WordPressKit
+
+class RemoteFeatureFlagStore {
+
+    init() {
+        DDLogInfo("🚩 Remote Feature Flag Device ID: \(deviceID)")
+    }
+
+    /**
+     Fetches remote feature flags from the server.
+
+     - Parameter callback: An optional callback that can be used to update UI following the fetch. It is not called on the UI thread.
+    */
+    public func update(then callback: FetchCallback? = nil) {
+        // We currently use an anonymous connection to WordPress.com, because we don't need user state.
+        // In the future, we could provide login credentials to the endpoint, which would allow customizing flags server-side on a per-user basis.
+        let remote = FeatureFlagRemote(wordPressComRestApi: WordPressComRestApi.defaultApi())
+        remote.getRemoteFeatureFlags(forDeviceId: deviceID) { [weak self] result in
+            switch result {
+                case .success(let flags):
+                    self?.cache = flags.dictionaryValue
+                    DDLogInfo("🚩 Successfully updated local feature flags: \(flags)")
+                    callback?()
+                case .failure(let error):
+                    DDLogError("🚩 Unable to update Feature Flag Store: \(error.localizedDescription)")
+            }
+        }
+    }
+
+    /// Checks if the local cache has a value for a given `FeatureFlag`
+    public func hasRemoteValueForFlag(_ flag: FeatureFlag) -> Bool {
+        return flag.remoteKey != nil && cache[flag.remoteKey!] != nil
```

Comment:
Let's deal with optionals early instead of force unwrapping

```suggestion
        guard let cachedKey = flag.remoteKey else {
            return false
        }

        return flag.remoteKey != nil && cache[cachedKey] != nil
```

Reply:
Addressed in c2015f3f1a05446e730105371d6f9b9d339ea7ba. I didn't use the commit suggestion because if we're using a `guard` statement we don't need the `flag.remoteKey != nil` check again in the `return` statement, so might as well get rid of it 🤷‍♂️